### PR TITLE
fix-11422 关系图边增加forceIgnore属性，不参加节点位置计算

### PR DESCRIPTION
**Pull request by TYzzt**
https://github.com/apache/incubator-echarts/issues/11422

### CODE REVIEW

File: src/chart/graph/forceHelper.js [hunk 1]
```diff
@@ -95,6 +95,9 @@ export function forceLayout(nodes, edges, opts) {
             var nLen = nodes.length;
             for (var i = 0; i < edges.length; i++) {
                 var e = edges[i];
+                if (e.forceIgnore) {
```
Comment (pissang) [inline src/chart/graph/forceHelper.js]: I suggested using `ignoreFoceLayout` instead of `forceIgnore`

File: src/data/Graph.js [hunk 2]
```diff
@@ -137,9 +137,10 @@ graphProto.getNodeById = function (id) {
  * @param {number|string|module:echarts/data/Graph.Node} n1
  * @param {number|string|module:echarts/data/Graph.Node} n2
  * @param {number} [dataIndex=-1]
+ * @param forceIgnore
  * @return {module:echarts/data/Graph.Edge}
  */
-graphProto.addEdge = function (n1, n2, dataIndex) {
+graphProto.addEdge = function (n1, n2, dataIndex, forceIgnore) {
```
Comment (pissang) [inline src/data/Graph.js]: The data used in forceHelper is initialized 

https://github.com/apache/incubator-echarts/blob/master/src/chart/graph/forceLayout.js#L90

There is no need to store this `ignoreFoceLayout` property in the Graph structure.

File: test/graph-force.html [hunk 3]
```diff
@@ -0,0 +1,130 @@
+
```
Comment (pissang) [inline test/graph-force.html]: Please use a more specific name to describe what test scene this is created for. Like force-edge-ignoreLayout.html

**Comment by pissang** [comment]
Hi @TYzzt . Thanks for your contribution. I've left the comments for change requests.

**Comment by TYzzt** [comment]
Thank you for your reply. I have modified the code

**Comment by TYzzt** [comment]
@pissang Will this feature be available in the next release?

**Comment by pissang** [comment]
@TYzzt Yes, it will be included in 4.5.0. Which will be released in the beginning of next month

**Pull request merged**